exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -112,18 +119,22 @@ class OTLPExporterMixin(ABC):
 
     def __init__(
         self,
-        endpoint: str = "localhost:55680",
+        endpoint: str,
+        insecure: bool,
         credentials: ChannelCredentials = None,
         metadata: tuple = None,
     ):
         super().__init__()
 
+        endpoint = endpoint or os.environ.get("OTEL_EXPORTER_OTLP_ENDPOINT") or "localhost:55680"
+        insecure = insecure or os.environ.get("OTEL_EXPORTER_OTLP_INSECURE") or True # TODO(jan25): verify spec for default value. This affects how credentials are passed
         self._metadata = metadata
         self._collector_span_kwargs = None
 
-        if credentials is None:
+        if insecure:
             self._client = self._stub(insecure_channel(endpoint))
         else:
+            credentials = credentials or _load_credential_from_file(os.environ.get("OTEL_EXPORTER_OTLP_CERTIFICATE"))
```

Comment:
If the file fails to load, what error message will the user see? Will it be helpful/obvious enough? Perhaps we shoudl wrap the exception with a more helpful and obvious error message. Something like, "Failed to load certificate from <path> as referenced by OTEL_EXPORTER_OTLP_CERTIFICATE"

Reply:
I've wrapped the file reading in a try block. Now it logs and returns `None` in case of exception.